elfpy/data/db_schema.py

```diff
@@ -46,6 +46,29 @@ class WalletInfo(Base):
     sharePrice: Mapped[Union[float, None]] = mapped_column(Numeric, default=None)
 
 
+class WalletDelta(Base):
```

Comment:
yeah this is a really weird schema the more i look at it.  don't bother with this until after the trading competition but 
`TokenDelta` makes more sense:

```python
class TokenDelta(Base):
    """Table/dataclass schema for token transfers."""

    __tablename__ = "tokendelta"

  se the Integer variant when using sqlite in tests
    id: Mapped[int] = mapped_column(
        BigInteger().with_variant(Integer, "sqlite"), primary_key=True, init=False, autoincrement=True
    )
    blockNumber: Mapped[int] = mapped_column(BigInteger, ForeignKey("poolinfo.blockNumber"), index=True)
    walletAddress: Mapped[Union[str, None]] = mapped_column(String, index=True, default=None)
    tokenAddress: Mapped[Union[str, None]] = mapped_column(String, index=True, default=None)
    tokenId: Mapped[Union[str, None]] = mapped_column(String, index=True, default=None)
    delta: Mapped[Union[float, None]] = mapped_column(Numeric, default=None)
    ```